kong/plugins/zipkin/handler.lua

```diff
@@ -12,7 +12,7 @@ function ZipkinLogHandler:new_tracer(conf)
 	local tracer = new_tracer(new_zipkin_reporter(conf), new_random_sampler(conf))
 	tracer:register_injector("http_headers", zipkin_codec.new_injector())
 	local function warn(str)
-		ngx.log(ngx.WARN, "[", self._name, "] ", str)
+		kong.log.warn("[", self._name, "] ", str)
```

Comment:
`"[", self._name, "] ", ` -> is not needed.

Reply:
it is actually needed, in fact. This is not the plugin's name, this is the *ZIPKIN REPORTER* name, which is a different kind of entity of which `kong.log` does know nothing about.